x/gov/client/rest/rest.go

```diff
@@ -202,6 +201,37 @@ func queryProposalHandlerFn(cdc *codec.Codec) http.HandlerFunc {
 	}
 }
 
+func queryDepositsHandlerFn(cdc *codec.Codec, cliCtx context.CLIContext) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		vars := mux.Vars(r)
+		strProposalID := vars[RestProposalID]
+
+		proposalID, ok := utils.ParseInt64OrReturnBadRequest(w, strProposalID)
+		if !ok {
+			return
+		}
+
+		params := gov.QueryDepositsParams{
+			ProposalID: proposalID,
+		}
+
+		bz, err := cdc.MarshalJSON(params)
```

Comment:
Do we want to optionally indent? I think there is a flag for that now.

Reply:
added